zhaquirks/xiaomi/aqara/motion_agl1.py

```diff
@@ -0,0 +1,194 @@
+"""Quirk for aqara lumi.sensor_occupy.agl1."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from zigpy import types
+from zigpy.quirks.v2 import (
+    NumberDeviceClass,
+    QuirkBuilder,
+    SensorDeviceClass,
+    SensorStateClass,
+)
+from zigpy.quirks.v2.homeassistant import EntityType, UnitOfLength
+from zigpy.zcl.clusters.general import DeviceTemperature
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.security import IasZone
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+from zhaquirks import LocalDataCluster
+from zhaquirks.xiaomi import XiaomiAqaraE1Cluster
+
+
+class AqaraMotion(types.enum8):
+    """Aqara motion attribute values."""
+
+    Idle = 0x02
+    Moving = 0x03
+    Still = 0x04
+
+
+class AqaraMotionSensitivity(types.enum8):
+    """Aqara motion sensitivity attribute values."""
+
+    Low = 0x01
+    Medium = 0x02
+    High = 0x03
+
+
+class AqaraOccupancy(types.enum8):
+    """Aqara occupancy attribute values."""
+
+    Unoccupied = 0x00
+    Occupied = 0x01
+
+
+class IasZoneLocal(LocalDataCluster, IasZone):
+    """Virtual cluster for IasZone."""
+
```

Comment:
So let's add the zone type as a constant attribute here, per the previous comment.
```suggestion

    _CONSTANT_ATTRIBUTES = {IasZone.AttributeDefs.zone_type.id: IasZone.ZoneType.Motion_Sensor}
```